src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -448,6 +450,7 @@ export default {
 	box-sizing: border-box;
 	width: 100%;
 	min-height: $clickable-area;
+	padding: 4px;
```

Comment:
@ma12-co This needs to be a `padding-right` only.

Reply:
I introduced that to give a bit of air to the hover circle of the actions and again balance to the whole thing :) 

Before:
![Screenshot from 2021-04-30 11-29-31](https://user-images.githubusercontent.com/26852655/116683593-bf5b5b00-a9a7-11eb-8d62-a9440f264aac.png)

After:
![Screenshot from 2021-04-30 11-28-34](https://user-images.githubusercontent.com/26852655/116683589-bec2c480-a9a7-11eb-85ef-174777f8d8b7.png)

Reply:
I often have the impression that we use quite a lot of white-space which sometimes wastes screen space. And I haven't had the feeling that more is necessary for AppNavigationItems (since e.g. the button background is only visible on hover). But then again, personal preference. If the others prefer it this way as well, fine with me.

Reply:
Maybe we could reduce it to two pixels on top and bottom, since with adjacent entries this sums up to twice the distance.